components/ui/search/SearchFilter.vue

```diff
@@ -45,7 +45,7 @@ export default {
   align-items: center;
   cursor: pointer;
   padding: 0.4rem 0.3rem;
-  margin: 3px 0 0 0.5rem;
+  margin: 3px 0.5rem 0 0.5rem;
```

Comment:
what's the point of this?

Reply:
It's part of an attempt to make the search filters panel spacing more uniform. A clearer way of doing this would be to move all the spacing CSS to the parent element. Moving the "clear filters" button to the bottom of the panel would also make things prettier.

Reply:
I removed all the spacing changes in 4c5f116